### ISSUE TITLE
Loading Alpinejs as a parcel in single-spa-react is failing 

### ISSUE THREAD

**Issue by tsukhu** [single-spa/single-spa-alpinejs#6]
Trying to load single-spa-alpinejs as a Parcel component in react caused a runtime error

```
single-spa-alpinejs.js:1 Uncaught (in promise) TypeError: During 'mount', parcel threw an error: parcel 'parcel-0' died in status NOT_MOUNTED: Converting circular structure to JSON
    --> starting at object with constructor 'HTMLDivElement'
    |     property '__reactInternalInstance$h6mr8ztska7' -> object with constructor 'FiberNode'
    --- property 'stateNode' closes the circle
    at JSON.stringify (<anonymous>)
    at VM15 single-spa-alpinejs.js:1
```

To reproduce use this codepen https://codepen.io/tsukhu/pen/QWyRXwo

**Issue:**
The React parcel is passing the domElement as part of props and when the alpinejs helper tries to serialize it and add it as x-data it fails

![ParcelOne](https://user-images.githubusercontent.com/846306/88781599-37db8400-d1aa-11ea-83a1-e6d4aee825f7.png)

![Parcel2](https://user-images.githubusercontent.com/846306/88781622-3ca03800-d1aa-11ea-8b9f-c0760173ddbe.png)

**Possible Solution**
Since we do not require the entire set of props to be passed to the alpinejs `x-data` we let the user decide which props they are interested in rather than serializing the entire props

```js
     // merge the opts.xData with the single-spa props
     // const finalData = Object.assign({}, props, originalData); <-- COMMENT OUT THIS LINE
    // Do not add all the props and just use the originalData as it will already have props added if the xData is a function and the user needs some specific prop they will destructure it from props and use
    ...
      // add x-data attribute
      childElement.setAttribute("x-data", JSON.stringify(originalData)); <-- PASS THE originalData that is resolved
```

**Comment by internettrans** [comment]
Also `singleSpa` is a prop that might need to be removed. I don't think using only the originalData is a good idea because it means it is not possible to pass in any custom props into the alpine application. Instead, maybe just add `delete finalProps.domElement`?

**Comment by internettrans** [comment]
Fixed in #7
